### PR TITLE
CRLF problem solved

### PR DESCRIPTION
**Pull request by tom1236868**
The CRLF problem is finally settle down, there is no conflict now, waited to be added and reply it ASAP.

### CODE REVIEW

**Comment by tom1236868** [comment]
The conflict has been solved, wait to be merged.

File: AUTHORS [hunk 1]
```diff
@@ -9,4 +9,5 @@ Elasql is written by:
   Yun-Sheng Chang <yschang@datalab.cs.nthu.edu.tw>
   Pin-Yu Wang <pywang@datalab.cs.nthu.edu.tw>
   Yu-Xuan Lin <yxlin@datalab.cs.nthu.edu.tw>
+  Chao-Wei Lin<cwlin@datalab.cs.nthu.edu.tw>
```
Comment (pin-yu) [inline AUTHORS]: please comply with the format

Comment (tom1236868) [reply AUTHORS]: The format error has been fix.

**Comment by pin-yu** [comment]
Looks good~

**Review by pin-yu — APPROVED** [review]
Good to go~

**Pull request merged**